docs/features/vpn.rst

```diff
@@ -32,8 +32,9 @@ increase throughput, although in practice the gain is minimal.
 
 **Site configuration:**
 
-1) Install ``gluon-web-mesh-vpn-fastd`` in ``site.mk``
+1) Add the package ``gluon-web-mesh-vpn-fastd`` in ``site.mk``
```

Comment:
as we would advise to use "features", please adjust your PR accordingly

Reply:
done